libs/model/src/community/GetCommunity.query.ts

```diff
@@ -10,20 +11,94 @@ export function GetCommunity(): Query<typeof schemas.GetCommunity> {
     secure: false,
     body: async ({ payload }) => {
       const where = { id: payload.id };
-      const include: Includeable[] = [];
+      const include: Includeable[] = [
+        {
+          model: models.CommunityBanner,
+        },
```

Comment:
this duplicates with `communityBanner` below

Reply:
Resolved -- we will query it separately for now. Can clean up with #8789